integr8ly/installation
# fix: handle walkthrough upgrade/install and restore old work queue/spring templates

## Additional Information
Fixes walkthrough 1A (and subsequent walkthroughs that depend on it)

## Verification Steps
As the verifier of the PR the following process should be done:

* Click "Get Started" for walkthrough 1a
* Verify that the project namespace "rhmi-lab-nodejs-order-ui" item deploys successfully

### Installation Verification
- Ensure the author of the PR has attached a log of the installation run from his branch to the jira or pr and check that it exited as expected.
- Verify the fresh installation is correct on cluster provided by PR author 

### Upgrade Verification
- After installation verification, notify the PR author to begin an upgrade on their cluster
- Ensure the developer of the PR has attached a log of the upgrade run from his branch to the jira or pr and check that it exited as expected. 
- If possible, look at the tasks that ran and see they match the PR
- Verify the upgrade is correct on cluster provided by PR author 

## Is an upgrade task required and are there additional steps needed to test this?
